lib/libutee/tee_api_operations.c

```diff
@@ -1418,11 +1420,6 @@ TEE_Result TEE_AEEncryptFinal(TEE_OperationHandle operation,
 		goto out;
 	}
 
-	if (!srcData && !srcLen) {
-		res = TEE_SUCCESS;
-		goto out;
-	}
-
```

Comment:
https://github.com/OP-TEE/optee_os/pull/1286 created. Separate test case isn't needed, it will be tested when this PR is merged.